workers/ohsome_quality_analyst/api/api.py

```diff
@@ -280,47 +280,47 @@ async def get_available_regions(asGeoJSON: bool = False):
         return response
 
 
-@app.get("/indicatorLayerCombinations")
-async def list_indicator_layer_combinations():
-    """List names of available indicator-layer-combinations."""
+@app.get("/indicators-layers")
+async def get_indicator_layer_combinations():
+    """Get names of available indicator-layer-combinations."""
     response = empty_api_response()
     response["result"] = INDICATOR_LAYER
     return response
 
 
-@app.get("/indicatorNames")
-async def list_indicators():
-    """List names of available indicators."""
+@app.get("/indicators")
```

Comment:
Why not keys?

Reply:
In the future I would like to return not just a list of keys but provide metadata about ervery indicator (e.g. description).
In this case do you think the term `indicators` is still to broad? I like the short and simple endpoint.
Should it be `indicator/keys` or `indicator-keys` or `indicator-metadata`?

Reply:
okay if this is the plan, then `/indicators` is fine